_component_design/bulk-selection.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Bulk Selection
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Bulk selection refers to the ways a user can simultaneously select and deselect multiple table rows. After bulk selection, the user can act on the group of selected objects. There are three types of bulk selection: keyboard, parent checkbox, and Select All and Deselect All.
+preview-image: preview-images/tables-controls.svg
+status: stable
+resource: true
+last-modified: 2018-12-11
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Add bulk selection options to a table when you want users to be able to select (or deselect) all visible rows, and then perform an action on the group of selected rows. If the information you are presenting in a table is read-only, do not provide bulk selection.
+
+When compared to selecting checkboxes individually, bulk selection is a more convenient approach to selecting or deselecting all visible rows.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you add bulk selection to a table:
+
+- The table should support the keyboard shortcuts to selecting multiple objects. For example, after selecting a row, the user can hold down the **Shift** key and select another row. This action selects all rows between the first and second selections.
```

Comment:
support keyboard accessibility (remove shortcuts)...remove 2nd and 3rd sentences.

Reply:
* Use Select and Deselect all so that the user doesn't need to select each row manually.
* Only select or deselect from the visible set of rows. Does not apply to all table pages.
* Use a selection bucket to manage multiple selections across pages.

Reply:
done.